README.md

```diff
@@ -343,6 +343,7 @@ Table of Contents
 ## Email
 
   * [mailinator.com](https://www.mailinator.com/) — Free, public, email system where you can use any inbox you want
+  * [cloudmersive.com](https://www.cloudmersive.com/email-verification-api) — Email validation and verification API for developers, 10,000 free API requests/month
```

Comment:
I do not see this free tier listed on the website under pricing information, where can I verify this exist?

Reply:
Sure - it is actually there on the pricing page as well as on the email verification page (the one linked) - if you scroll down, it is in a black horizontal bar that says "10,000 free API calls/month, no credit card required, no expiration".  In that bar there is a Create Account button and also a Google SSO button - both of which will allow you to create an account and then once logged in, you can create an API key - which you will see has 10,000 free calls per month.  The 10,000 calls per month does not expire, it will continue to work without expiration.

You can see it on these two pages among others:

https://www.cloudmersive.com/pricing
https://www.cloudmersive.com/email-verification-api

You can also access it at any time by clicking the Sign Up button in the navigation bar as well.  You can see when you click Sign Up on the right-hand side the free tier details are listed, including 10,000 API calls/per month, etc.

Does that answer your question?

Reply:
Yes. Weird I thought I even ^f searched for “free” on there :) must be going crazy